BootloaderCorePkg/Stage2/Stage2.c

```diff
@@ -214,6 +216,7 @@ NormalBootPath (
     DEBUG ((DEBUG_INFO, "FV Format Payload\n"));
     UefiSig = Dst[0];
     Status  = LoadFvImage (Dst, Stage2Param->PayloadActualLength, (VOID **)&PldEntry, &PldMachine);
+#if !FixedPcdGetBool (PcdElfSupportDisabled)
   } else if (IsElfFormat ((CONST UINT8 *)Dst)) {
```

Comment:
Line 219 uses the preprocessor directive, it could build the ELF code out at compile time. With below code it is more readable and it could also build ELF code out when PcdElfSupportEnabled is FALSE
 } else if (FeaturePcdGet (PcdElfSupportEnabled) && IsElfFormat ((CONST UINT8 *)Dst)) {

Reply:
Updated